sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_aggregates_v2/query.sql

```diff
@@ -0,0 +1,84 @@
+SELECT
+  submission_date,
+  app_version,
+  os,
+  channel,
+  locale,
+  browser_version,
+  country,
+  newtab_homepage_category,
```

Comment:
Since this is a newtab table, I'd recommend avoiding the use of preffix `newtab`, unless you think it can be confused with other data.

Reply:
We have a few places where we use the `newtab_` prefix.
examples:
- `newtab_visit_id`
- `newtab_homepage_category`
- `newtab_newtab_category`
- `newtab_search_enabled`

@florezita @jaredsnyder any thoughts about not using the `newtab_` prefix?

Reply:
Great point. Once we agree to a naming convention, we'd need to update downstream dependencies, as part of the implementation.

Reply:
I have renamed only two fields:
`newtab_newtab_category` to `newtab_category`
`newtab_homepage_category` to `homepage_category`